tests/e2e/tests/basic/e2e.ts

```diff
@@ -34,6 +34,21 @@ export default function () {
     .then(() => copyFile('./e2e/renamed-app.e2e-spec.ts', './e2e/another-app.e2e-spec.ts'))
     .then(() => ng('e2e', '--specs', './e2e/renamed-app.e2e-spec.ts',
       '--specs', './e2e/another-app.e2e-spec.ts'))
+    // Suites block need to be added in the protractor.conf.js file to test suites
+    .then(() => replaceInFile('protractor.conf.js', `allScriptsTimeout: 11000,`,
```

Comment:
I think this is a good test. I tried to do this without the your PR and e2e broke with `Error: Unknown test suite: app`.

Reply:
Thank you :). Yap, I also tried same and some other combination and thought to add suites and then test it.